src/js/dropdown-menu/dropdown.underscore

```diff
@@ -9,13 +9,9 @@
 </a>
 <button type="button" class="menu-button button-more has-dropdown js-dropdown-button <% if (!button.icon) { %>default-icon<% } %>" aria-haspopup="true" aria-expanded="false" aria-controls="<%- menuId %>">
     <% if (button.icon) { %>
-        <span class="icon-fallback icon-fallback-img">
-            <span class="icon <%- button.icon %>" aria-hidden="true"></span>
-            <span class="text"><%- button.label %></span>
-        </span>
-    <% } else { %>
-        <span class="sr-only"><%- button.label %></span>
+        <span class="icon fa <%- button.icon %>" aria-hidden="true"></span>
```

Comment:
Do we want to assume that we can only use Font Awesome icons? I suppose that's probably reasonable.